cli/deploy-profiling-job.sh

```diff
@@ -0,0 +1,123 @@
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
+## Please reach out to the Azure ML docs & samples team before before editing for the first time.
+
+## Preparation Steps:
+## 1. az upgrade -y
+## 2. az extension remove -n ml
+## 3. az extension remove -n azure-cli-ml
+## 4. az extension add -n ml
+## 5. az login
+## 6. az account set --subscription "<YOUR_SUBSCRIPTION>"
+## 7. az configure --defaults group=<RESOURCE_GROUP> workspace=<WORKSPACE_NAME>
+
+# <set_variables>
+export SUBSCRIPTION="<SUBSCRIPTION>"
+export RESOURCE_GROUP="<RESOURCE_GROUP>"
+export WORKSPACE_NAME="<WORKSPACE_NAME>"
+export ENDPOINT_NAME="<ENDPOINT_NAME>"
+export DEPLOYMENT_NAME="<DEPLOYMENT_NAME>"
+export PROFILING_TOOL="<PROFILING_TOOL>"
```

Comment:
list all supported profiling tools

Reply:
added comment for listing out all allowed values for PROFILING_TOOLS